packages/note/src/react/index.js

```diff
@@ -0,0 +1,214 @@
+import * as glamor from 'glamor'
```

Comment:
As we look forward to using ES modules in order to tree shake, I think we should start using named imports specifically where we are not "for sure going to use everything in this package".  This is because, as I understand it, if you `import * from`, nothing in that imported package can be tree-shaken.

I recently attempted this in datawell: https://github.com/pluralsight/design-system/blob/master/packages/datawell/src/react/index.js#L1

What do you think?

Reply:
i'm ok with that going forward 👍